Makefile

```diff
@@ -35,9 +35,13 @@ cmake-debug:
 debug: cmake-debug
 	cd build/debug && $(MAKE)
 
+# Temporarily disable some a tests:
+#  * coretests takes too long (~25000s)
+#  * libwallet_api_tests fail (Issue #895)
+#  * unit_tests were run separately above
 debug-test:
 	mkdir -p build/debug
-	cd build/debug && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Debug ../.. && $(MAKE) && $(MAKE) test
+	cd build/debug && cmake -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=Debug ../.. &&  $(MAKE) && $(MAKE) ARGS="-E coretests\|libwallet_api_tests" test
```

Comment:
remove the backslash (no linee continuation here) and add single quotes (to make sure absolutely shell doesn't interpret the pipe) like so: `... ARGS="-E 'coretests|lib_wallet_api....'"  ...`

Reply:
coretests removed. no pipe needed.  backslash removed.